GCRCatalogs/cosmodc2.py

```diff
@@ -139,6 +139,7 @@ def _subclass_init(self, catalog_root_dir, catalog_filename_template, **kwargs):
         self.sky_area, self._native_quantities, self._quantity_info = self._process_metadata(**kwargs)
         self._quantity_modifiers = self._generate_quantity_modifiers()
         self._native_filter_quantities = {'healpix_pixel', 'redshift_block_lower'}
+        self.halo_mass_def = "FoF, b=0.168"
```

Comment:
Thanks @dkorytov. Let's allow overwriting this attribute when specified config file, for potential future use. 

```suggestion
        self.halo_mass_def = kwargs.get('halo_mass_def', 'FoF, b=0.168')
```